### PR TITLE
Replace now-defunct UM6 as default, use CV5-25 instead, as-per RPSW-661

### PR DESCRIPTION
**Pull request by civerachb-cpr**
Part 2 of purging references to UM6 from the repo, as per RPSW-661

### CODE REVIEW

**Review by tonybaltovski — CHANGES_REQUESTED** [review]
There are still Heron's that use this IMU, I would not want to remove support for it.  Perhaps consider adding support for new IMUs by default?

**Comment by civerachb-cpr** [comment]
Had a discussion with Dave & Jason about this.  The plan is to just fork the MSCL driver + dependencies and release them from our own build farm, since Lord doesn't seem inclined to do it themselves.  This should resolve the outstanding issues with incorporating the new driver as the default.

File: heron_base/launch/base.launch [hunk 1]
```diff
@@ -26,26 +26,40 @@
...
-     <node pkg="um6" type="um6_driver" name="um6_driver" output="screen">
-       <param name="tf_ned_to_enu" value="true"/>
-       <param name="/um6_driver/port" value="$(optenv HERON_IMU_UM6_PORT /dev/clearpath/imu)" />
-       <remap from="imu/data" to="imu/data_raw" />
-       <param name="lt_model" value="true" />
-       <param name="linear_acceleration_stdev" value="0.38" />
-       <param name="frame_id" value="imu_link" />
-     </node>
+    <group unless="$(optenv HERON_IMU_UM6 0)">
+      <!-- newer herons use the microstrain CM5-25 IMU -->
```
Comment (tonybaltovski) [inline heron_base/launch/base.launch:30]: Should there be a case if there is a namespace?

File: heron_base/launch/base.launch [hunk 2]
```diff
@@ -26,26 +26,40 @@
...
+      <include filename="$(find ros_mscl)/launch/microstrain.launch">
+        <arg name="name"          value="cv5" />
+        <arg name="port"          value="$(optenv HERON_IMU_PORT /dev/microstrain)" /> <!-- TODO - verify the correct default -->
```
Comment (tonybaltovski) [inline heron_base/launch/base.launch]: Verify please

File: heron_base/launch/base.launch [hunk 3]
```diff
@@ -26,26 +26,40 @@
...
+        <arg name="imu_frame_id"  value="imu_link" />
+        <!-- TODO - double-check topic names -->
```
Comment (tonybaltovski) [inline heron_base/launch/base.launch]: Verify please

**Review by tonybaltovski — CHANGES_REQUESTED** [review]

File: heron_base/launch/base.launch (hunk 1 above)
Comment (civerachb-cpr) [reply heron_base/launch/base.launch:30]: Possibly. I'm looking into that along with the 2 points below

File: heron_base/launch/base.launch (hunk 3 above)
Comment (civerachb-cpr) [reply heron_base/launch/base.launch]: Same as the other note to myself; the PR is WIP, but I'll address any memos to myself before final review.

File: heron_base/launch/base.launch (hunk 1 above)
Comment (civerachb-cpr) [reply heron_base/launch/base.launch:30]: Added the namespace case, checked the topics & default device as mentioned in the other notes.

**Review by tonybaltovski — APPROVED** [review]

**Pull request merged**